cli/src/pcluster/resources/imagebuilder/parallelcluster_validate.yaml

```diff
@@ -237,8 +237,14 @@ phases:
 
                   echo "Checking Intel MPI 20xx installed and module available..."
                   unset MODULEPATH
-                  source /etc/profile.d/modules.sh
-                  (module avail intelmpi)2>&1 | grep "/opt/intel/mpi/20.*/modulefiles/"
+                  if [[ -f /etc/profile.d/modules.sh ]] ; then
+                    source /etc/profile.d/modules.sh
+                  elif [[ -f /etc/profile.d/00-modulepath.sh ]] ; then
```

Comment:
can both `/etc/profile.d/modules.sh` and  `/etc/profile.d/00-modulepath.sh` coexist?
I'm wondering if we need to source them all if they are present and then change the check for the failure when no one of them is present.